docs/README.md

```diff
@@ -1,5 +1,4 @@
-[![logo](https://aws-ia.github.io/assets/img/taskcat-bg.png)](https://aws-ia.github.io/)
-__https://aws-ia.github.io__
+[![logo](https://raw.githubusercontent.com/aws-quickstart/taskcat/main/assets/docs/images/tcat.png)](https://taskcat.io/)
```

Comment:
We have migrated taskcat to the aws-ia org.  Some changes are needed to the image path in your PR